app/notification/model/notification.py

```diff
@@ -72,6 +72,18 @@ def email_recipient(json_data: dict):
                     template_name=NotifyConstants.TEMPLATE_TYPE_EOI_PASS_W_CAVEATS,
                 )
 
+            case NotifyConstants.TEMPLATE_TYPE_ASSESSMENT_APPLICATION_ASSIGNED:
```

Comment:
Have we created the template in notification service (GOV Notify)

Reply:
I think that was done by others as part of the ticket, thanks for the prompt though we'll make sure its in -- I used some IDs I found in the ticket here https://github.com/communitiesuk/funding-service-design-notification/pull/258/files#diff-830b41638f32c4afb3ccef5bf4c2c26af86371a2e52b088f895297c5577bca63R92-R99